maritalk/model.py

```diff
@@ -0,0 +1,77 @@
+import requests
+from typing import List, Dict, Union
+
+
+class MariTalk:
+    def __init__(self, key: str, api_url: str = "https://chat.maritaca.ai/api"):
+        self.key = key
+        self.api_url = api_url
+
+    def generate(
+        self,
+        messages: Union[str, List[Dict[str, str]]],
```

Comment:
@ZanezZephyrs, maybe `input` is a better name?
It might be a bit confusing to use "messages when the input is a string (eg.`messages="oi"`)

Reply:
Even though `input` is a built-in function, I agree that it's a better naming and it would only affect the scope of this method anyway.